magefiles/development.go

```diff
@@ -133,6 +133,21 @@ func (d Dev) Integration(ctx context.Context) error {
 	return nil
 }
 
+func (d Dev) IntegrationRun(ctx context.Context, testName string) error {
+	mg.SerialDeps(
+		Dev.Deploy,
+	)
+
+	os.Setenv("KUBECONFIG", devEnvironment.Cluster.Kubeconfig())
+	os.Setenv("ENABLE_WEBHOOK", "true")
+	os.Setenv("ENABLE_API_MOCK", "true")
+	os.Setenv("ENABLE_PROMETHEUS_REMOTE_STORAGE_MOCK", "true")
+	os.Setenv("EXPERIMENTAL_FEATURES", "true")
+
+	mg.SerialDeps(mg.F(Test.IntegrationRun, testName))
+	return nil
+}
+
```

Comment:
This won't do much good, we're duplicating an existing functionality here.

We can't really set and persist env vars for a parent process; it is true only during periodic runtime.